arangod/Scheduler/Scheduler.cpp

```diff
@@ -303,30 +366,41 @@ void Scheduler::addQueueStatistics(velocypack::Builder& b) const {
   b.add("in-progress", VPackValue(static_cast<int32_t>(numWorking(counters))));
   b.add("queued", VPackValue(static_cast<int32_t>(numQueued(counters))));
   b.add("queue-size", VPackValue(static_cast<int32_t>(_maxQueueSize)));
-  b.add("current-fifo1", VPackValue(static_cast<int32_t>(_fifoSize[0])));
-  b.add("fifo1-size", VPackValue(static_cast<int32_t>(_maxFifoSize[0])));
-  b.add("current-fifo2", VPackValue(static_cast<int32_t>(_fifoSize[1])));
-  b.add("fifo2-size", VPackValue(static_cast<int32_t>(_maxFifoSize[1])));
+  b.add("current-fifo1", VPackValue(static_cast<int32_t>(_fifoSize[FIFO1])));
```

Comment:
all the `static_cast<int32_t>` conversions should be removed here, because we don't need them at all.

Reply:
done